util-images/containerd/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM docker:latest
```

Comment:
Do we need docker in this image?

Reply:
Oh didn't we want to maintain backward compatibility with docker?

Reply:
Nah, let's make it work only for containerd. It's already a default and docker is being deprecated. If people want to use this feautre with docker they can use older Cl2 release. In case there is a need to support both docker and containerd we can always add some gating logic in the daemonset to use either docker image & docker command or the new containerd image and ctr command. An image should support only one solution, either ctr or docker.

Reply:
Got it, if that's the case I'll use a minimal base image and create a separate PR for that